hyppo/ksample/energy.py

```diff
@@ -3,7 +3,8 @@
 from ._utils import _CheckInputs
 from .base import KSampleTest, KSampleTestOutput
 from .ksamp import KSample
-
+import numpy as np
+import dcor
```

Comment:
I would prefer not adding `dcor` as a dependency to this package as it would be redundant. We already have methods to compute Dcorr and Energy (see respective doc strings and classes for references)